### PR TITLE
Alpine variant of Dockerfile

### PR DESCRIPTION
**Pull request by trulede**
Hello, in case you are interested, I have made an Alpine variant of your Dockerfile. The resultant image is somewhat smaller (160MB against 250MB). Also, to get either image to build I needed a few small changes which I have included as individual commits.

### CODE REVIEW

**Comment by swilly22** [comment]
Hi @trulede,
Thank you for this PR,
may I ask what's the reason for switching to Alpine? is that purely image size?

**Comment by trulede** [comment]
Hi, 

yes this is partly to get a smaller image size. We use alpine variants of most images (i.e. Redis) as well as the basis for our own images, therefore it's also a little easier to manage.

However, please don't feel obliged to take the PR if it's not something you see fitting to your project - which by the way is a most curious thing. I use Neo4j together with Redis for dynamic (throwaway) systems and wonder about using RedisGraph as an alternative because Neo4j is a bit "heavy".

**Comment by swilly22** [comment]
@trulede do you know who's maintaining redis:5-alpine?
can you link to dockerhub?
How quickly is it updated when a new version of Redis is released?

**Comment by trulede** [comment]
Hi,

I take all my Redis images from the "official Docker" repo at : https://hub.docker.com/_/redis/ and they seem to follow the latest Redis releases (versions 5 & 6) without much delay. The git repo is here : https://hub.docker.com/_/redis/ .

File: Dockerfile.alpine [hunk 1]
```diff
@@ -0,0 +1,44 @@
+###
+# Build Stage.
+FROM redis:5-alpine as builder
```
Comment (jeffreylovitz) [inline Dockerfile.alpine:3]: One change request - can we use `redis:6-alpine` as the base image? We are generally migrating in that direction, so we could be proactive by adopting it here!

Comment (trulede) [reply Dockerfile.alpine:3]: That's fine by me. Let me push one other small change, just a moment.

**Pull request merged**